x-pack/plugins/security_solution/server/features.ts

```diff
@@ -143,16 +144,17 @@ export const getKibanaPrivilegesFeaturePrivileges = (ruleTypes: string[]): Kiban
       ui: ['show', 'crud'],
     },
     read: {
-      app: [APP_ID, 'kibana'],
+      app: [APP_ID, 'csp', 'kibana'],
       catalogue: [APP_ID],
-      api: [APP_ID, 'lists-read', 'rac'],
+      api: [APP_ID, 'lists-read', 'rac', 'csp-read'],
```

Comment:
Please rename `csp-read` to `cloud-security-posture-read` - `csp` has semantic meaning in Kibana